docs/Client_Server.md

```diff
@@ -0,0 +1,39 @@
+# Client and Server
+
+## Precompile
+
+Server side we precompile all our React components into static HTML.
+
+Use the `npm install` command to trigger or update the precompile.
+
+### Webpack
+
+Webpack creates static HTML pages from a list of react routes.
```

Comment:
Ah I see you mentioned that in line 19, but I think you might want to talk about it first here.